packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -128,6 +128,7 @@ class ImagePicker {
     double? maxWidth,
     double? maxHeight,
     int? imageQuality,
+    int? limit,
```

Comment:
The new `limit` parameter should be documented in the doc-block above, since this function is directly used by end users of the plugin, and they might want to [know what it does](https://pub.dev/documentation/image_picker/1.0.7/image_picker/ImagePicker/pickMultiImage.html). 

(This is also a good place to remark that on the web this parameter is ignored!)